src/sentry/workflow_engine/tasks/delayed_workflows.py

```diff
@@ -1,10 +1,26 @@
 from __future__ import annotations
 
-from typing import Any
+from collections import defaultdict
```

Comment:
it'd be nice if we can decompose this file a bit - it's a little hard to grok the task that is being added here and how it's being processed -- splitting those two concepts into two files might make it a bit easier.

Reply:
I copy pasted the task from `delayed_workflow` in `processors` into here